Tests/RevenueCatUITests/CustomerCenter/ManageSubscriptionsViewModelTests.swift

```diff
@@ -71,27 +71,187 @@ class ManageSubscriptionsViewModelTests: TestCase {
         expect(viewModel.isLoaded) == true
     }
 
-    func testLoadScreenSuccess() async {
+    func testShouldShowActiveSubscription_whenUserHasOneActiveSubscriptionOneEntitlement() async {
+        // Arrange
+        let productId = "com.revenuecat.product"
+        let purchaseDate = "2022-04-12T00:03:28Z"
+        let expirationDate = "2062-04-12T00:03:35Z"
+        let products = [Fixtures.product(id: productId, title: "title", duration: .month, price: 2.99)]
+        let customerInfo = Fixtures.customerInfo(
+            subscriptions: [
+                Fixtures.Subscription(id: productId, store: "app_store", purchaseDate: purchaseDate, expirationDate: expirationDate)
+            ],
+            entitlements: [
+                Fixtures.Entitlement(entitlementId: "premium", productId: productId, purchaseDate: purchaseDate, expirationDate: expirationDate)
+            ]
+        )
+        
         let viewModel = ManageSubscriptionsViewModel(screen: ManageSubscriptionsViewModelTests.screen,
-                                                     purchasesProvider: MockManageSubscriptionsPurchases())
+                                                     purchasesProvider: MockManageSubscriptionsPurchases(
+                                                        customerInfo: customerInfo,
+                                                        products: products
+                                                     ))
 
+        // Act
         await viewModel.loadScreen()
 
+        // Assert
         expect(viewModel.subscriptionInformation).toNot(beNil())
         expect(viewModel.screen).toNot(beNil())
         expect(viewModel.state) == .success
 
         expect(viewModel.subscriptionInformation?.title) == "title"
         expect(viewModel.subscriptionInformation?.durationTitle) == "month"
         expect(viewModel.subscriptionInformation?.price) == "$2.99"
-        expect(viewModel.subscriptionInformation?.nextRenewalString) == "Apr 12, 2062"
-        expect(viewModel.subscriptionInformation?.productIdentifier) == "com.revenuecat.product"
+        
+        let formatter = DateFormatter()
+        formatter.dateStyle = .medium
+        expect(viewModel.subscriptionInformation?.nextRenewalString) == formatter.string(from: ISO8601DateFormatter().date(from: expirationDate)!)
```

Comment:
This fixed this test running on my local machine, where my locale would not have formatted the date as `"Apr 12, 2062"`.